vulnerable_people_form/form_pages/shared/validation.py

```diff
@@ -295,6 +297,31 @@ def validate_phone_number_if_present(section_key, phone_number_key):
     return True
 
 
+def validate_sms_phone_number_if_present(section_key, phone_number_key, logger=None):
+    phone_number = form_answers()["contact_details"].get(phone_number_key, "")
+    if phone_number and phone_number_is_valid_for_notify(phone_number, logger=logger):
+        return True
+    else:
+        error_message = "Enter a UK mobile number, like 07700 900000 or +44 7700 900000"
+        error_section = session.setdefault("error_items", {}).get(section_key, {})
+        session["error_items"] = {
+            **session.setdefault("error_items", {}),
+            section_key: {**error_section, phone_number_key: error_message},
+        }
+        return False
+
+
+def phone_number_is_valid_for_notify(phone_number, logger=None):
+    try:
+        validate_notify_compatible_uk_mobile_number(phone_number)
+        return True
+    except InvalidPhoneError as e:
+        if logger:
+            logger.warning(create_log_message(log_event_names["NHS_LOGIN_USER_CONSENT_NOT_GIVEN"],
```

Comment:
i don't this constant is right - create new one for this scenario

Reply:
I've made a new one and put it in appropriately